src/components/AlertModal.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
```

Comment:
If we are no longer using `AlertModal` can we delete it?

Reply:
Resolved in https://github.com/isomerpages/isomercms-frontend/pull/247/commits/bdee7c02a8e8f625df3f997e060beb7808e11425